Tests/New-HipchatUser.Tests.ps1

```diff
@@ -1,16 +1,32 @@
 ﻿Import-Module PHipChatAdmin
 
 Describe "New-HipchatUser" {
-
-    Context "When the API call is sent" {
-
-        It "Should return a 201 Status Code" {
-            New-HipchatUser -FirstName 'Pester' -LastName 'Test' -ApiToken $ApiToken | 
-            Should Be '201'
+    InModuleScope PHipChatAdmin {
+        
+        Context "When the API call is sent" {
+            
+            It 'Attempts to invoke web api' {
+                Mock Invoke-WebRequest -MockWith {
+                    $returned = @{'id' = '123456';
+                                'Content' = 'someContent';
+                                'Status' = 'someStatus';
+                                'StatusCode' = '201'
+                                }
+                    return $returned
+                }
+                $global:result = New-HipchatUser -FirstName 'Pester' -LastName 'Test' -ApiToken 'REXsCauSe553gsoIJg1Gj4zwNsSAwS'
+                Assert-MockCalled -CommandName 'Invoke-WebRequest' -Times 1 -Scope It
+            }
+            
+            It 'StatusCode field exists and not null' {
+                $result.StatusCode.ToString() | Should Not Be $null
```

Comment:
Found the issue - this should be $Result.ToString() instead of $Result.StatusCode.ToString(). The reason for this is that the function already returns the StatusCode.

Reply:
The Mock Invoke-WebRequest should return a hash table with keys id, Content, Status, and StatusCode. If $result is just a single string with a return code, I think it means my $global:result line is not using the mocked Invoke-WebRequest. Whatcha think?